packages/@lwc/engine-server/package.json

```diff
@@ -26,12 +26,14 @@
     "//": {
         "parse5": "Currently can't upgrade parse5: dropped cjs support in v7.0.0 https://github.com/inikulin/parse5/releases/tag/v7.0.0"
     },
+    "dependencies": {
+        "parse5": "^6.0.1"
+    },
```

Comment:
parse5 is now used inside of `@lwc/engine-server` at runtime (not just during testing), so it now needs to be a regular dep, not a devDep.